pages/search.js

```diff
@@ -237,6 +254,15 @@ export default function Search({
                   onClick={() => handleSearchTag(tag.name)}
                 />
               ))}
+
+          <button
```

Comment:
Can we use our button component `components/Button.js`?

Reply:
I have replaced it with our Button component.